interfaces/Portalicious/src/app/components/query-table/query-table.component.ts

```diff
@@ -146,6 +146,11 @@ export class QueryTableComponent<TData extends { id: PropertyKey }, TContext> {
   readonly contextMenu = viewChild<Menu>('contextMenu');
   readonly extraOptionsMenu = viewChild<Menu>('extraOptionsMenu');
 
+  selectedColumnsStateKey = computed(() => {
+    const key = this.localStorageKey();
+    return key ? `${key}-selected-columns` : 'selected-columns';
```

Comment:
```suggestion
    return key ? `${key}-selected-columns` : undefined;
```

I think if we don't have a key to save the filters, then we shouldn't save the selected columns either.